src/module-elasticsuite-catalog/view/adminhtml/layout/search_term_edit.xml

```diff
@@ -14,10 +14,11 @@
  */
  -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <update handle="smile_elasticsuite_upsell_notification"/>
```

Comment:
Seems to create a problem when creating a new search term.
When submitting the form, it redirects to `/admin/search/term/new/key/[key]/**undefined**`.

Reply:
Fixed, I let you check again